src/web/docusaurus/docs/getting-started/environment-setup.md

```diff
@@ -288,7 +300,14 @@ Run the `pnpm db:init` script as it will populate the database with an initial s
 `db:seed` is responsible to populate the database with an initial set of data.
 :::
 
-Stop the `parser` service running. If you have Docker app installed locally make sure to stop the `parser` container before continuing to the next step.
+Docker Desktop app:
+![Original Telescope architecture](../static/img/initialProjectDiagram.png)
```

Comment:
Is this supposed to display the image or is it just a link?
If it's a link, I don't think we want to use the text "Original Telescope architecture" for it.

Reply:
So I should change it back to Image?

Reply:
Ok ill change that